### PR TITLE
many improvements, fix docs

### PR DESCRIPTION
**Pull request by ignaigna**
title, some of the improvements:

- fix README.md example
- follow proper flake8 rules
- cleanup code

### CODE REVIEW

File: rblxopencloud/creator.py [hunk 1]
```diff
@@ -254,55 +264,66 @@ def upload_asset(
 
         payload = {
             "assetType": (
-                asset_type.name if type(asset_type) == AssetType
-                else asset_type
+                asset_type.name if type(asset_type) == AssetType else asset_type
```
[the remark this answers is not in the document]
Comment (ignaigna) [reply rblxopencloud/creator.py:267]: black formatter does it like that, maybe doing a custom config would fix this properly?

File: rblxopencloud/creator.py [hunk 2]
```diff
@@ -385,56 +412,59 @@ def list_asset_versions(
...
-            "POST", f"assets/v1/assets/{asset_id}/versions:rollback",
-            authorization=self.__api_key, expected_status=[200],
-            json={
-                "assetVersion": f"assets/{asset_id}/versions/{version_number}"
-            }
+            "POST",
+            f"assets/v1/assets/{asset_id}/versions:rollback",
+            authorization=self.__api_key,
+            expected_status=[200],
+            json={"assetVersion": f"assets/{asset_id}/versions/{version_number}"},
```
Comment (treeben77) [inline rblxopencloud/creator.py:467]: >79 character

Comment (ignaigna) [reply rblxopencloud/creator.py:467]: > black formatter does it like that, maybe doing a custom config would fix this properly?

**Review by treeben77 — APPROVED** [review]

**Comment by LawMixer** [comment]
treeben7 forgot to merge my pull request in, so sorry for any of the merge conflicts!

**Comment by treeben77** [comment]
ill let @LawMixer review if they want to before merging

**Pull request merged**